spacewire/libgrspw/libgrspw.c

```diff
@@ -544,13 +544,18 @@ static int spw_rxConfigure(spw_dev_t *dev, size_t *firstDesc, const size_t nPack
 
 
 /* Read from RX buffers */
-static int spw_rxRead(spw_dev_t *dev, size_t firstDesc, uint8_t *buf, size_t bufsz, const size_t nPackets)
+static int spw_rxRead(spw_dev_t *dev, uint8_t *buf, size_t bufsz, size_t *val, const spw_t *ictl)
```

Comment:
Maybe some better variable name than `val`, e.g. `readCnt`?
```suggestion
static int spw_rxRead(spw_dev_t *dev, uint8_t *buf, size_t bufsz, size_t *readCnt, const spw_t *ictl)
```